specifyweb/frontend/js_src/lib/components/SchemaConfig/UniquenessRules.tsx

```diff
@@ -0,0 +1,674 @@
+                relationships.find(({ name }) => name === field)
+              )?.localization.name as string
+            }
+          />
+        ))}
+      </td>
+      <td>
+        <Input.Text
+          disabled
+          value={
+            rule.scopes.length === 0
+              ? schemaText.database()
+              : getFieldsFromPath(model, rule.scopes[0])
+                  .map((field) => field.localization.name as string)
+                  .join(' -> ')
+          }
+        />
+        {isModifyingRule && (
+          <ModifyUniquenessRule
+            fetchedDuplicates={fetchedDuplicates}
+            fields={fields}
+            label={label}
+            model={model}
+            readOnly={readOnly}
+            relationships={relationships}
+            rule={rule}
+            onChange={handleChanged}
+            onClose={toggleModifyingRule}
+            onRemoved={handleRemoved}
+          />
+        )}
+      </td>
+    </tr>
+  );
+}
+
+function ModifyUniquenessRule({
+  rule,
+  model,
+  readOnly,
+  label,
+  fields,
+  relationships,
+  fetchedDuplicates,
+  onChange: handleChanged,
+  onRemoved: handleRemoved,
+  onClose: handleClose,
+}: {
+  readonly rule: UniquenessRule;
+  readonly model: SpecifyModel;
+  readonly readOnly: boolean;
+  readonly label: string;
+  readonly fields: RA<LiteralField>;
+  readonly relationships: RA<Relationship>;
+  readonly fetchedDuplicates: UniquenessRuleValidation | undefined;
+  readonly onChange: (newRule: typeof rule) => void;
+  readonly onRemoved: () => void;
+  readonly onClose: () => void;
+}): JSX.Element {
+  const [separator] = userPreferences.use(
+    'queryBuilder',
+    'behavior',
+    'exportFileDelimiter'
+  );
+
+  const uniqueFields = React.useMemo(
+    () =>
+      [...fields, ...relationships].map(
+        (item) => [item.name, item.localization.name as string] as const
+      ),
+    [fields, relationships]
+  );
+
+  return (
+    <Dialog
+      buttons={
+        <>
+          <Button.Danger
+            disabled={!hasPermission('/schemaconfig/uniquenessrules', 'delete')}
+            onClick={(): void => {
+              handleRemoved();
+              handleClose();
+            }}
+          >
+            {commonText.delete()}
+          </Button.Danger>
+          <span className="-ml-2 flex-1" />
+          {fetchedDuplicates !== undefined &&
+          fetchedDuplicates.totalDuplicates > 0 ? (
+            <Button.Danger
+              onClick={(): void => {
+                const fileName = `${model.name} ${rule.fields
+                  .map((field) => field)
+                  .toString()}-in_${
+                  rule.scopes.length === 0
+                    ? schemaText.database()
+                    : getFieldsFromPath(model, rule.scopes[0])
+                        .map((field) => field.name)
+                        .join('_')
+                }.csv`;
+
+                const columns = [
+                  'Duplicate Values',
```

Comment:
localize this

Reply:
Fixed in https://github.com/specify/specify7/pull/4215/commits/43aee5b97e52edca55b58e0cb2b76773754b510f

Reply:
not fixed?